opentelemetry-auto-instrumentation/README.rst

```diff
@@ -0,0 +1,32 @@
+OpenTelemetry Auto Instrumentation
+============================================================================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-api.svg
+   :target: https://pypi.org/project/opentelemetry-auto-instrumentation/
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-auto-instrumentation
+
+References
+----------
+
+* `OpenTelemetry Project <https://opentelemetry.io/>`_
+
+Usage
+-----
+
+This package provides a command that automatically instruments a program:
+
+::
+
+    opentelemetry-auto-instrumentation program.py
+
+The code in ``program.py`` needs to use one of the packages for which there is
+an OpenTelemetry extension. For a list of the available extensions please check
+`here <https://opentelemetry-python.readthedocsio/>`_.
```

Comment:
It's better to use an "internal" target for this link. So if we decide to host our documentation somewhere else the generated link is not broken.

Reply:
Makes sense, but what do you mean with "internal"? Can you provide an example, please?